apps/console/src/features/organizations/api/organization.ts

```diff
@@ -334,8 +333,7 @@ export const shareApplication = (
             "Content-Type": "application/json"
         },
         method: HttpMethods.POST,
-        url: `${store.getState().config.endpoints.organizations}/organizations/${currentOrganizationId}/applications/` +
```

Comment:
Can we be sure that this would not affect legacy mode i.e cloud offering behaviour?

Reply:
Fixed